### PR TITLE
chore: Cleanup double words

### PR DESCRIPTION
**Pull request by nschonni**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
Thanks again @nschonni 

This is nice.

Did you find a way to automate this, or are you finding and fixing by hand?

**Comment by nschonni** [comment]
Just manual with https://github.com/errata-ai/vale to highlight the issues. I think there are some that need to be suppressed because they are intentional to show some of the RegEx examples

File: docs/framework/data/adonet/sql/handling-null-values.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ WHERE TerritoryID IN (1, 2, 3)
   
  In addition, the following rules apply for an instance of `DataRow.["columnName"]` null assignments:  
   
-1. The default *default* value is `DbNull.Value` for all except the strongly typed null columns where it is the appropriate strongly typed null value.
```
Comment (gewarren) [inline docs/framework/data/adonet/sql/handling-null-values.md:91]: This one was possibly intentional, given the italicization of the second word. I can see there being a default default value.

Comment (nschonni) [reply docs/framework/data/adonet/sql/handling-null-values.md:91]: True, I can't think of a better way of expressing it though if it was intentional 😦

**Pull request merged**